apps/functions-subscription/src/domain/activation-job.ts

```diff
@@ -27,3 +33,43 @@ export const ActivationJobCodec = t.strict({
 });
 
 export type ActivationJob = t.TypeOf<typeof ActivationJobCodec>;
+
+export type InsertActivationJob = Pick<
```

Comment:
```suggestion
type InsertActivationJob = Pick<
```
suggestion(non-blocking): it's not used somewhere else